airflow/models.py

```diff
@@ -56,8 +56,8 @@
 
 from sqlalchemy import (
     Column, Integer, String, DateTime, Text, Boolean, ForeignKey, PickleType,
-    Index, Float, LargeBinary, UniqueConstraint)
-from sqlalchemy import func, or_, and_, true as sqltrue
+    Index, Float, LargeBinary, UniqueConstraint, ForeignKeyConstraint)
```

Comment:
small nit: it would be good to keep the import list sorted(given you are touching on this line :))

Reply:
Hm, the line is already not sorted (Index, Float) ;). Should I then sort the whole 3-line import statement?